context.go

```diff
@@ -133,6 +135,14 @@ func (blipCtx *Context) GetBytesReceived() uint64 {
 	return blipCtx.bytesReceived.Load()
 }
 
+// Opens a BLIP connection to a host.
```

Comment:
This comment doesn't make sense, but see below where I think we should force passing a context when `NewContext` is specified. I think we could fail when you call `BlipContext.Dial`.